Intermittent build error in Syntax Generator tooling

Sometimes when building Roslyn clean (after git clean -xdf) I get these errors:

                   "F:\SourcesToIndex\Roslyn\Roslyn.sln" (Rebuild target) (1) ->
                   "F:\SourcesToIndex\Roslyn\Open\src\ExpressionEvaluator\VisualBasic\Source\ResultProvider\Portable\BasicResultProvider.Portable.vbproj" (Rebuild target) (97:2) ->
                   (GenerateSyntaxModelGetText target) -> 
                     F:\SourcesToIndex\Roslyn\Open\build\Targets\GenerateCompilerInternals.targets(83,5): error MSB3073: The command ""F:\SourcesToIndex\Roslyn\Open\Binaries\Debug\VBSyntaxGenerator.exe" "..\..\..\..\..\Compilers\VisualBasic\Portable\Syntax\Syntax.xml" "F:\SourcesToIndex\Roslyn\Open\Binaries\Obj\BasicResultProvider.Portable\Debug\\Syntax.xml.Generated.vb" /gettext" exited with code 9009. [F:\SourcesToIndex\Roslyn\Open\src\ExpressionEvaluator\VisualBasic\Source\ResultProvider\Portable\BasicResultProvider.Portable.vbproj]


                   "F:\SourcesToIndex\Roslyn\Roslyn.sln" (Rebuild target) (1) ->
                   "F:\SourcesToIndex\Roslyn\Open\src\ExpressionEvaluator\VisualBasic\Source\ResultProvider\NetFX20\BasicResultProvider.NetFX20.vbproj" (Rebuild target) (121) ->
                     F:\SourcesToIndex\Roslyn\Open\build\Targets\GenerateCompilerInternals.targets(83,5): error MSB3073: The command ""F:\SourcesToIndex\Roslyn\Open\Binaries\Debug\NetFX20\..\VBSyntaxGenerator.exe" "..\..\..\..\..\Compilers\VisualBasic\Portable\Syntax\Syntax.xml" "F:\SourcesToIndex\Roslyn\Open\Binaries\Obj\BasicResultProvider.NetFX20\Debug\\Syntax.xml.Generated.vb" /gettext" exited with code 9009. [F:\SourcesToIndex\Roslyn\Open\src\ExpressionEvaluator\VisualBasic\Source\ResultProvider\NetFX20\BasicResultProvider.NetFX20.vbproj]

--- a/src/ExpressionEvaluator/VisualBasic/Source/ResultProvider/NetFX20/BasicResultProvider.NetFX20.vbproj
+++ b/src/ExpressionEvaluator/VisualBasic/Source/ResultProvider/NetFX20/BasicResultProvider.NetFX20.vbproj
@@ -39,6 +39,11 @@
       <Project>{BEDC5A4A-809E-4017-9CFD-6C8D4E1847F0}</Project>
       <Name>ResultProvider.NetFX20</Name>
     </ProjectReference>
+    <ProjectReference Include="..\..\..\..\..\Tools\Source\CompilerGeneratorTools\Source\VisualBasicSyntaxGenerator\VisualBasicSyntaxGenerator.vbproj">
+      <Project>{6AA96934-D6B7-4CC8-990D-DB6B9DD56E34}</Project>
+      <Name>VisualBasicSyntaxGenerator</Name>
+      <ReferenceOutputAssembly>false</ReferenceOutputAssembly>
+    </ProjectReference>
   </ItemGroup>
   <ItemGroup Label="Linked Files">
     <Compile Include="..\..\..\..\..\Compilers\VisualBasic\Portable\Scanner\CharacterInfo.vb">

--- a/src/ExpressionEvaluator/VisualBasic/Source/ResultProvider/Portable/BasicResultProvider.Portable.vbproj
+++ b/src/ExpressionEvaluator/VisualBasic/Source/ResultProvider/Portable/BasicResultProvider.Portable.vbproj
@@ -37,6 +37,11 @@
       <Project>{fa0e905d-ec46-466d-b7b2-3b5557f9428c}</Project>
       <Name>ResultProvider.Portable</Name>
     </ProjectReference>
+    <ProjectReference Include="..\..\..\..\..\Tools\Source\CompilerGeneratorTools\Source\VisualBasicSyntaxGenerator\VisualBasicSyntaxGenerator.vbproj">
+      <Project>{6AA96934-D6B7-4CC8-990D-DB6B9DD56E34}</Project>
+      <Name>VisualBasicSyntaxGenerator</Name>
+      <ReferenceOutputAssembly>false</ReferenceOutputAssembly>
+    </ProjectReference>
   </ItemGroup>
   <ItemGroup Label="Linked Files">
     <Compile Include="..\..\..\..\..\Compilers\VisualBasic\Portable\Scanner\CharacterInfo.vb">

